main.go

```diff
@@ -68,14 +69,21 @@ func main() {
 			"profiler-addr", profilerAddress)
 		go runProfiler(profilerAddress)
 	}
+
+	// Default webserver port is 9443.
 	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: metricsAddr,
-		LeaderElection:     enableLeaderElection,
-		Port:               9443,
```

Comment:
huh looks like this field was for the webhook port originally but that was not enabled, odd that it was configured at all

Reply:
We do have a webhook https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/blob/main/api/v1alpha1/akodeploymentconfig_webhook.go
I think we just used the default config

Reply:
```
// WebhookServer is an externally configured webhook.Server. By default,
	// a Manager will create a default server using Port, Host, and CertDir;
	// if this is set, the Manager will use this server instead.
	WebhookServer *webhook.Server
```